pkg/jobs/create_latest_distributions.go

```diff
@@ -0,0 +1,69 @@
+package jobs
+
+import (
+	"context"
+	"sync"
+
+	"github.com/content-services/content-sources-backend/pkg/api"
+	"github.com/content-services/content-sources-backend/pkg/config"
+	"github.com/content-services/content-sources-backend/pkg/dao"
+	"github.com/content-services/content-sources-backend/pkg/db"
+	"github.com/content-services/content-sources-backend/pkg/pulp_client"
+	"github.com/content-services/content-sources-backend/pkg/tasks/helpers"
+	"github.com/rs/zerolog/log"
+)
+
+func CreateLatestDistributions() {
+	err := db.Connect()
+	if err != nil {
+		log.Fatal().Err(err).Msg("failed to connect to database")
+	}
+
+	daoReg := dao.GetDaoRegistry(db.DB)
+	ctx := context.Background()
+
+	domains, err := daoReg.Domain.List(ctx)
+	if err != nil {
+		log.Fatal().Err(err).Msg("failed to list domains")
+	}
+	for _, domain := range domains {
+		pulpClient := pulp_client.GetPulpClientWithDomain(domain.DomainName)
+		distHelper := helpers.NewPulpDistributionHelper(ctx, pulpClient)
+
+		pageData := api.PaginationData{Limit: -1}
+		filterData := api.FilterData{Origin: config.OriginExternal + "," + config.OriginUpload}
```

Comment:
i'm seeing 404s for the latest snapshot urls in RH repos, looks like RH repos aren't included here. was that intentional?

Reply:
I guess I was thinking red hat repos won't really have issue because they get updated frequently. But it probably makes sense to include them anyway! Just in case.